distributed/widgets/__init__.py

```diff
@@ -0,0 +1 @@
+from .utils import get_environment, get_template  # noqa
```

Comment:
I don't think this is needed as `flake8` is configured to ignore `__init__.py` files

```suggestion
from .utils import get_environment, get_template
```

Reply:
Done!